examples/typescript/src/locations/Sidebar.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { PlainClientAPI } from 'contentful-management';
 import { Paragraph } from '@contentful/f36-components';
 import { SidebarExtensionSDK } from '@contentful/app-sdk';
+import { useCMA, useSDK } from "@contentful/react-apps-toolkit";
 
-interface SidebarProps {
-  sdk: SidebarExtensionSDK;
-  cma: PlainClientAPI;
-}
+const Sidebar = () => {
+  const sdk = useSDK<SidebarExtensionSDK>()
+  /* You can remove the next line as soon as you use the CMA, or, if not needed, simply remove the hook. */
+  /*  eslint-disable @typescript-eslint/no-unused-vars */
+  const cma = useCMA();
```

Comment:
This pattern of having to disable some lint rule in every file feels a bit weird. Could we maybe just have the  `const cma = useCMA();` itself commented out?

Reply:
+1, I also think it's not ideal to sip a `eslint-disable` with the template. But having a commented `cma` constant is also strange...

If we leave it, it should be `eslint-disable-next-line`.